firebase-appdistribution/src/test/java/com/google/firebase/appdistribution/FirebaseAppDistributionTest.java

```diff
@@ -480,22 +481,43 @@ public void updateIfNewReleaseAvailable_whenScreenRotates_updateDialogReappears(
 
     // Mimic activity recreation due to a configuration change
     firebaseAppDistribution.onActivityDestroyed(activity);
-    firebaseAppDistribution.onActivityCreated(activity);
+    firebaseAppDistribution.onActivityResumed(activity);
 
     assertAlertDialogShown();
     assertFalse(updateTask.isComplete());
   }
 
   @Test
-  public void taskCancelledOnDestroy_whenSignInDialogShowing() {
+  public void
+      updateIfNewReleaseAvailable_signInTaskCancelled_whenSignInDialogShowingAndNewActivityStarts() {
```

Comment:
Nit: for this and the next test, could you change the name of the test to put the "when" clause in the middle of the name, and the expected behavior at the end? E.g. `updateIfNewReleaseAvailable_whenSignInDialogShowingAndNewActivityStarts_signInTaskCancelled`